### ISSUE TITLE
Mermaid diagrams are cropped

### ISSUE THREAD

**Issue by jcayzac** [hiroppy/fusuma#391]
## Info

- Operating System: macOS 11.2
- Node Version: 15.6.0
- Browser Name and Version: Chrome 88
- Fusuma Version: 2.5.0

## Type

- [x] bug
- [ ] feature request

## For Bugs

With the default theme, mermaid diagrams that are larger than 320px are cropped.

<img width="1516" alt="Screen Shot 2021-02-18 at 18 16 45" src="https://user-images.githubusercontent.com/106682/108334974-cdf1fd00-7215-11eb-9a2c-66ecf9cda577.png">

### Expected Behavior

Diagrams aren't cropped but resized instead.

### Actual Behavior

Diagrams are cropped.

### How can we reproduce the behavior?

Test diagram:

```
gitGraph:
options
&#x7b;
  &#x22;nodeSpacing&#x22;: 150,
  &#x22;nodeRadius&#x22;: 10
&#x7d;
end
commit
branch newbranch
checkout newbranch
commit
commit
checkout master
commit
commit
merge newbranch
```

**Comment by hiroppy** [comment]
Fixed, thanks. https://hiroppy.github.io/fusuma/advanced/#slide-1

**Comment by jcayzac** [comment]
@hiroppy thank you. It still looks cropped on my phone tho (the SVG doesn't seem to have CSS to constrain its width to the viewport):

![Screenshot_20210218_220438_com android chrome](https://user-images.githubusercontent.com/106682/108361387-c2163300-7235-11eb-9659-dca3c5c82733.jpg)

**Comment by hiroppy** [comment]
yeah, I think we should add `overflow` to the container class. I'll consider it.

**Comment by hiroppy** [comment]
hm, I checked it but mermaid calculates the current client size and yes it doesn' have `viewport`....
